docs/en/wallet-solution.rst

```diff
@@ -5,55 +5,58 @@
 
-To guarantee the utmost security, these cryptographic keys are securely stored within the device's Trusted Execution Environment (TEE)[3]. This ensures that only the User is allowed to access them, thus preventing unauthorized usage or tampering. For more detailed information, please refer to the `Wallet Attestation section`_ and the `Trust Model section`_ of this document.
+To guarantee the utmost security, these cryptographic keys MUST be securely stored within the WSCD which MAY be internal (device's Trusted Execution Environment (TEE)[3]), external, or hybrid. This ensures that only the User is allowed to access them, thus preventing unauthorized usage or tampering. For more detailed information please refer to the `Wallet Attestation section`_ and the `Trust Model section`_ of this document.
 
 Wallet Instance Lifecycle
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 The Wallet Instance has three distinct states: Operational, Valid, and Deactivated. Each state represents a specific functional status and determines the actions that can be performed[2].
 
 Initialization Process
 ~~~~~~~~~~~~~~~~~~~~~~
-To activate the Wallet Instance, the Users MUST install the mobile wallet application on their device and open it. Furthermore, Users will be asked to set their preferred method of unlocking their device; this can be accomplished by entering a personal identification number (PIN) or by utilizing biometric authentication, such as fingerprint or facial recognition, according to their personal preferences and device's capabilities.
+To activate the Wallet Instance, the Users MUST install the mobile Wallet application on their device and open it. Furthermore, Users will be asked to set their preferred method of unlocking their device; this can be accomplished by entering a personal identification number (PIN) or by utilizing biometric authentication, such as fingerprint or facial recognition, according to their personal preferences and device's capabilities.
 
 After completing these steps, the Wallet Instance sets the Operational state.
 
 Transition to Valid state
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 To transition from the Operational state to the Valid state, the Wallet Instance MUST obtain a valid Personal Identification (PID). Once a valid PID is acquired, the Wallet Instance becomes Valid.
 
-In order to securely and unambiguously identify Users, the Wallet Instance adopts a Level of Assurance (LoA) 3 authentication, which guarantees a high level of confidence in the User's identity. The authentication method is chosen by the PID Provider from among the notified eID solutions at the national level.
+In order to securely and unambiguously identify Users, the Wallet Instance requires a Level of Assurance (LoA) 3, which guarantees both the User's identity and the device security. The authentication method is chosen by the PID Provider from among the notified eID solutions at the national level.
```

Comment:
The LoA is not related to the device security. The LoA only refers to the identification process. I would differentiate the LoA from the AAL. The latter is related to the device security. 
There is some confusion on this point:
- National eIDs (eIDAS notified) have a LoA that is related to the identification procedures and authentication mechanisms --> this doesn't involve the Wallet Instance
- The LoA in the context of the EUDI Wallet is dependent on the security of the Wallet Instance itself. 


I see 2 options:
1. we use 2 different concepts: LoA (for user identification during the issuance) and AAL (for the security of the device and the SW)
2. we introduce a LoA specific for the EUDI Wallet which includes both (LoA for the user identification and AAL for the security of the device). This means that if the User requests a Credential using national eID with LoA High, if the Wallet Instance doesn't provide any evidence of AAL High, the Credential will not be High (to be define all the cases, eg. LoA id = High + AAL=Law --> LoA EUIDW=???).

**Side Note**: in the Credential there is no evidence about the LoA. We only have an evidence of identity assurance in the credential definition and it is the same for all Credentials of the type